### DISCUSSION TITLE
Endless downloads in the "Recent repositories" and "Profile" section on github

### DISCUSSION THREAD

**Discussion by rprudchenko** [community/community#38499] [Other Feature Feedback, Questions, & Ideas] [answered]
I am currently using safari version 12.1.2 (Mac Os Mojave), and there are endless downloads in the Recent Repositories and Profiles section (there is a phrase "Sorry, something went wrong" in the Profiles section), can someone explain to me why this is happening?

**Comment by david-engelmann** [comment]
It sounds like the page is having an issue. Does the problem persist if you refresh the page or try incognito mode?

**Reply by rprudchenko** [reply]
Yes, I refreshed the page and checked the incognito mode, these problems still exist. I uploaded the video with problems to Google drive https://drive.google.com/file/d/11NUKsGr2Bz1eVdHfNryf0A7WP4qSjKlW/view?usp=sharing

**Reply by david-engelmann** [reply]
Are you using a VPN?

**Reply by rprudchenko** [reply]
No, I am not

**Reply by david-engelmann** [reply]
Interesting, it seems like the request is being rejected for some reason, does inspecting the page show any errors?

**Reply by rprudchenko** [reply]
I opened the browser console and saw 21 problems, all of these problems report a "Syntax error: unexpected token'.'" in different files.
I created a folder and uploaded to it a text file containing all the problems in my browser console and a video demonstrating these problems https://drive.google.com/drive/folders/1cvdzU6G7xt6j4n3anGyctBJdx_J66gr2?usp=sharing

**Comment by david-engelmann** [comment, marked as answer]
Totally an issue processing the request. I'd upgrade safari to 14.1.2, it seems like Stripe may be causing an issue, either way the root cause looks like some elements of Github are not supported by the older version!
